src/lib/components/data-graphic/utils.ts

```diff
@@ -1,6 +1,10 @@
 import { line, area, curveLinear, curveStep } from "d3-shape";
 import type { ScaleLinear, ScaleTime } from "d3-scale";
 import type { GraphicScale } from "./state/types";
+import { get, writable } from "svelte/store";
+
+export const breakpointX = writable(150);
```

Comment:
Why use writables here? Are we expecting to change these values in a way that needs to propagate throughout component sets?

If so, we should be setting these differently.

An easy starting point would be to just make these `const`

Reply:
ah, I see why.

Reply:
I would maybe not opt to have these breakpoints determined here like this, instead maybe passed in as a prop to the axis